mmv1/products/artifactregistry/terraform.yaml

```diff
@@ -33,6 +33,25 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           kms_key_name: "kms-key"
         test_vars_overrides:
           kms_key_name: 'BootstrapKMSKeyInLocation(t, "us-central1").CryptoKey.Name'
+      - !ruby/object:Provider::Terraform::Examples
+        name: "artifact_registry_repository_virtual"
+        min_version: beta
+        primary_resource_id: "my-repo"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-my-repository%s\", context[\"random_suffix\"])"
+        vars:
+          repository_id: "my-repository"
+          description: "example virtual docker repository"
+          upstream_repository_id: "my-repository-upstream"
+          upstream_description: "example docker repository (upstream source)"
+          upstream_policy_id: "my-repository-upstream"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "artifact_registry_repository_remote"
+        min_version: beta
+        primary_resource_id: "my-repo"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-my-repository%s\", context[\"random_suffix\"])"
```

Comment:
It looks like this variable primary_resource_name is not used and can be removed.

Reply:
Removed.